easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1706-foss-2016a.eb

```diff
@@ -0,0 +1,44 @@
+# -*- python -*-
+name = 'OpenFOAM'
+version = 'v1706'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a GPL-opensource C++ CFD-toolbox.
+This offering is not approved or endorsed by OpenCFD Ltd,
+producer and distributor of the OpenFOAM software via www.openfoam.com,
+and owner of the OPENFOAM trademark.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
```

Comment:
There's little point in providing easyconfigs for an older toolchain like `foss/2016a` imho.

If people really need to install OpenFOAM with it, they can use `--try-toolchain`.

Generally speaking, it's better to stick to more recent toolchains, so I'd prefer just having easyconfigs for `foss/2016b` here.